src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -272,7 +272,7 @@ BtreeLBAManager::alloc_extent(
 	    c,
 	    *state.insert_iter,
 	    state.last_end,
-	    lba_map_val_t{len, addr, 1, 0},
+	    lba_map_val_t{len, pladdr_t(addr), 1, 0}
```

Comment:
should have " , " after lba_map_val_t{len, pladdr_t(addr), 1, 0}.  fix the bug in this commit.